packages/blade/.storybook/react/global.css

```diff
@@ -0,0 +1,4 @@
+* {
+  /* Need this to ensure mdx stories don't break visually*/
+  box-sizing: border-box;
```

Comment:
Storybook changes this randomly, have to set it so that this doesn't keep changing